instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/__init__.py

```diff
@@ -35,12 +35,19 @@
     cnx.close()
 
     pool = await aiopg.create_pool(database='Database')
+    
     cnx = await pool.acquire()
     cursor = await cnx.cursor()
     await cursor.execute("INSERT INTO test (testField) VALUES (123)")
     cursor.close()
     cnx.close()
-
+    
+    cnx = AiopgInstrumentor().instrument_connection(cnx)
```

Comment:
For this and the other helpful `Usage` examples you've added, please could we also add something so that readers know they don't have to do both `instrument` and `instrument_connection` calls?

It could be like this:

```suggestion
    # Alternatively, use instrument_connection for an individual connection
    cnx = AiopgInstrumentor().instrument_connection(cnx)
```

Or, it could be two separate `.. code-block:: python`. The first block would be for `# Call instrument() to wrap all database connections` and the second block would be for `# Alternatively, use instrument_connection for an individual connection`

Reply:
Makes sense. Updated docstring to have two separate `. code-block:: python` blocks

Reply:
Looks good! Please could you make the same update for mysql, mysqlclient, psycopg, psycopg2, pymysql, sqlite3